nautilus_trader/adapters/interactive_brokers/client/market_data.py

```diff
@@ -390,7 +391,8 @@ async def get_historical_bars(
                 return []
             self._log.debug(f"reqHistoricalData: {request.req_id=}, {contract=}")
             request.handle()
-            return await self._await_request(request, timeout, default_value=[])
+            ib_timeout = int(os.environ.get("IB_REQUEST_TIMEOUT", timeout))
```

Comment:
Hi @marcodambros

If you can remove these lines sourcing the timeout from an env var, we can then merge this and I'll add a config option for the timeout.